source/locale/en/symbols.dic

```diff
@@ -82,7 +82,6 @@ _	line	most
 •	bullet	some
 …	dot dot dot	all	always
 ...	dot dot dot	all	always
-	bullet	some

```

Comment:
Because it is a PUA character and as such it will only be a bullet in certain applications. For Word this character is now mapped to a proper bullet.